packages/k6-tdk/src/auth/kopano.ts

```diff
@@ -0,0 +1,132 @@
+import { fail } from 'k6'
+import http from 'k6/http'
+import { get } from 'lodash-es'
+
+import { cleanURL, objectToQueryString, queryStringToObject, randomString } from '@/utils'
+
+import { Authenticator, Token } from './auth'
+
+export class Kopano implements Authenticator {
```

Comment:
well, or stick withOpenIDConnect with config options for the login url, username and passwond params ... ? that might be enough for a generic openidconnect auth

Reply:
wont't work :( i can have a look in the next oidc pr